booking-app/src/main/resources/webapp/js/app.js

```diff
@@ -21,7 +21,7 @@
             })
             .when('/bookingApp/primeVaccination', {
                 templateUrl: '../booking-app/resources/partials/primeVaccination.html',
-                controller: 'BookingAppPrimeVaccinationCtrl'
+                controller: 'BAPrimeVaccinationCtrl'
```

Comment:
I think we should use one naming in all controllers, for me BookingApp looks better than BA, @kkaczmarczyk what do you think?